camunda-formio-tasklist-vue/src/services/axios.ts

```diff
@@ -12,6 +12,18 @@ export function bpmAxios (bearerToken: string, CamundaUrl: string) {
   });
 }
 
+export function bpmAxiosWithHal (bearerToken: string, CamundaUrl: string) {
+  return axios.create({
+    baseURL: CamundaUrl,
+    withCredentials: false,
+    headers: {
+      "Access-Control-Allow-Origin": "*",
+      "Content-Type": "application/hal+json",

```

Comment:
Isn't Accept to be changed to `application/hal+json`? When running the taskList, I am getting 415 error.

![image](https://user-images.githubusercontent.com/70306694/139676894-b099ddab-e903-40ca-a6de-454d5f96f275.png)

Reply:
Corrected.